src/main/java/mops/presentation/DirectoryController.java

```diff
@@ -118,7 +117,7 @@ public String deleteFolder(KeycloakAuthenticationToken token,
     public String searchFolder(KeycloakAuthenticationToken token,
                                Model model,
                                @PathVariable("dirId") long dirId,
-                               @ModelAttribute("searchQuery") FileQuery query) {
+                               @RequestAttribute("searchQuery") FileQuery query) {
```

Comment:
Can you explain why `@RequestAttribute` I thought this isn't working. Not critic just curious.